src/main/java/org/scijava/module/AbstractModule.java

```diff
@@ -76,11 +78,16 @@ public void cancel() {
 	public void initialize() throws MethodCallException {
 		// execute global module initializer
 		final Object delegateObject = getDelegateObject();
-		if (initializerRef == null) {
-			final String initializer = getInfo().getInitializer();
-			initializerRef = new MethodRef(delegateObject.getClass(), initializer);
+		if (delegateObject instanceof Initializable) {
+			((Initializable) delegateObject).initialize();
+		}
+		else {
```

Comment:
There is also a question of handling the case where the `Module` itself is `Initializable`. This can happen e.g. for modules which extend `AbstractModule` directly. I think we should check that instead, and then have `CommandModule` implement `Initializable`, working analogously to [how support works for `Cancelable`](https://github.com/scijava/scijava-common/blob/scijava-common-2.60.2/src/main/java/org/scijava/command/CommandModule.java#L208-L233).